integration/build_and_launch_test.go

```diff
@@ -0,0 +1,94 @@
+package integration_test
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/paketo-buildpacks/occam"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+	. "github.com/paketo-buildpacks/occam/matchers"
+)
+
+func testBuildAndLaunch(t *testing.T, context spec.G, it spec.S) {
```

Comment:
Nitpick: I think instead of calling this the build and launch test, this is more of the simple app case. This name is slighly misleading considering this is a test for the basic behaviour of the buildpack. If you rename the test, I'd rename the fixture too.

Reply:
it's called `default_app` now